lib/lighthouse/benefits_claims/service.rb

```diff
@@ -37,6 +43,9 @@ def get_claims(lighthouse_client_id = nil, lighthouse_rsa_key_path = nil, option
 
     def get_claim(id, lighthouse_client_id = nil, lighthouse_rsa_key_path = nil, options = {})
```

Comment:
Btw, get claims doesn’t receive any tracked item data so there’s not much point in trying to remove them here